NGRAVE ZERO brand import list integration

caiquecruz opened the pull request:
## NGRAVE ZERO brand import list integration

Note:
- Since Keystone already uses the `bc-ur` package and NGRAVE ZERO shares the same device type `QR Hardware Wallet Device`, explicit checks have been added to ensure proper handling.

caiquecruz commented on src/ui/views/HDManager/AccountList.tsx:
```diff
@@ -177,7 +181,7 @@ export const AccountList: React.FC<Props> = ({
             .find((item) => item.type === keyring);
           const alias = generateAliasName({
-            brandName: brandName,
+            brandName: isNgraveZero ? 'NGRAVE ZERO' : brandName,
```
@heisenberg-2077  Without this explicit check, it would display as 'Keystone' because it shares the same keyring type: 'QR Hardware Wallet Device', can we do this for this specific case ?

vvvvvv1vvvvvv replied:
We need to modify the code to ensure that other brands that also use QRCode keyring will not be displayed as keystone. Please wait for us to modify it again to ensure that you do not need to do `isNgraveZero` recognition anywhere separately.

caiquecruz commented:
> Please sync the latest code from the develop branch, add the NGRAVE ZERO as follows
> 
> https://github.com/RabbyHub/Rabby/blob/develop/src/ui/views/NewUserImport/ImportList.tsx#L68
> 
> ```tsx
>         {
>           type: KEYRING_CLASS.HARDWARE.KEYSTONE,
>           logo: WALLET_BRAND_CONTENT[WALLET_BRAND_TYPES.NGRAVEZERO].icon,
>           brand: WALLET_BRAND_CONTENT[WALLET_BRAND_TYPES.NGRAVEZERO].brand,
>         },
> ```

@heisenberg-2077 should I update it ? And also remove the isNgraveZero checks ?

caiquecruz commented:
> > > Please sync the latest code from the develop branch, add the NGRAVE ZERO as follows
> > > [`develop`/src/ui/views/NewUserImport/ImportList.tsx#L68](https://github.com/RabbyHub/Rabby/blob/develop/src/ui/views/NewUserImport/ImportList.tsx?rgh-link-date=2025-04-08T12%3A38%3A38Z#L68)
> > > ```tsx
> > >         {
> > >           type: KEYRING_CLASS.HARDWARE.KEYSTONE,
> > >           logo: WALLET_BRAND_CONTENT[WALLET_BRAND_TYPES.NGRAVEZERO].icon,
> > >           brand: WALLET_BRAND_CONTENT[WALLET_BRAND_TYPES.NGRAVEZERO].brand,
> > >         },
> > > ```
> > 
> > 
> > @heisenberg-2077 should I update it ? And also remove the `isNgraveZero` checks ?
> 
> Yes, just add the wallet brand to the tipList array and test to see if it works as expected.

@heisenberg-2077 worked perfectly, thank you! Also reverted the explicit checks and removed the unused i18n keys.
